curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -214,6 +214,28 @@ FSStatusCode MdsClientImpl::GetFsInfo(uint32_t fsId, FsInfo *fsInfo) {
     return ReturnError(rpcexcutor_.DoRPCTask(task, mdsOpt_.mdsMaxRetryMS));
 }
 
+bool MdsClientImpl::CommitTx(const std::vector<PartitionTxId>& txIds) {
```

Comment:
MdsClientImpl interface should return appropriate return code, not only true or false, like FSStatusCode (maybe called MdsStatusCode better).

Reply:
> MdsClientImpl interface should return appropriate return code, not only true or false, like FSStatusCode (maybe called MdsStatusCode better).

fixed.